org.eclipse.jdt.launching/plugin.xml

```diff
@@ -278,10 +278,6 @@
             description="%environment.description.10"
             id="OSGi/Minimum-1.2"
             ruleParticipant="org.eclipse.jdt.internal.launching.environments.DefaultAccessRuleParticipant"/>            
-      <environment
-            description="%environment.description.2"
```

Comment:
Could you please also delete the description key/value from properties

Reply:
Fixed.

Reply:
- https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/schema/executionEnvironments.exsd#L202 .

- https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/environments/ExecutionEnvironmentAnalyzer.java#L67

- https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/7c57761f182f30e385b7c980f1f80390d75eadc2/org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/environments/ExecutionEnvironment.java#L495

I would have a look if those can be safely removed as well.

Reply:
I have removed it.

Reply:
Could you push a PR with this nevertheless, to see which / how many tests will fail?

Reply:
I have updated the PR. It should work now.